cocos/particle-2d/tiff-reader.ts

```diff
@@ -460,127 +465,127 @@ export class TiffReader {
-                        case 0:
-                            let invertValue = 0;
-                            if ((sampleProperties[0] as any).hasBytesPerSample) {
-                                invertValue = Math.pow(0x10, (sampleProperties[0] as any).bytesPerSample * 2);
-                            }
-
-                            // Invert samples.
-                            pixelSamples.forEach((sample, index, samples): void => {
-                                samples[index] = invertValue - sample;
-                            });
-
-                            // Bilevel or Grayscale
-                            // BlackIsZero
-                        case 1:
-                            red = green = blue = this.clampColorSample(pixelSamples[0], (sampleProperties[0] as any).bitsPerSample);
-                            break;
-
-                            // RGB Full Color
-                        case 2:
-                            red = this.clampColorSample(pixelSamples[0], (sampleProperties[0] as any).bitsPerSample);
-                            green = this.clampColorSample(pixelSamples[1], (sampleProperties[1] as any).bitsPerSample);
-                            blue = this.clampColorSample(pixelSamples[2], (sampleProperties[2] as any).bitsPerSample);
-                            break;
-
-                            // RGB Color Palette
-                        case 3:
-                            if (colorMapValues === undefined) {
-                                throw Error(getError(6027));
-                            }
-
-                            const colorMapIndex = pixelSamples[0];
-
-                            red = this.clampColorSample(colorMapValues[colorMapIndex], 16);
-                            green = this.clampColorSample(colorMapValues[colorMapSampleSize + colorMapIndex], 16);
-                            blue = this.clampColorSample(colorMapValues[(2 * colorMapSampleSize) + colorMapIndex], 16);
-                            break;
+                    switch (photometricInterpretation) {
+                    // Bilevel or Grayscale
+                    // WhiteIsZero
+                    case 0:
+                    {
+                        let invertValue = 0;
+                        if (sampleProperties[0].hasBytesPerSample) {
+                            invertValue = 0x10 ** (sampleProperties[0].bytesPerSample! * 2);
+                        }
 
-                            // Unknown Photometric Interpretation
-                        default:
-                            throw RangeError(getError(6028, photometricInterpretation));
+                        // Invert samples.
+                        pixelSamples.forEach((sample, index, samples): void => {
+                            samples[index] = invertValue - sample;
+                        });
+                    }
+                    // Bilevel or Grayscale
+                    // BlackIsZero
+                    case 1:
```

Comment:
Please fix eslint error.

Reply:
Could not fix it since we don't know whether it was designed like that.
The lint said `case 0` should have a `break` but I don't know whether it was designed like that or a mistake.
This is the code first committed in `cocos2d-html5`.

Reply:
And it's not this PR cause the eslint issue.

Reply:
Added comment and ignore the eslint check.